sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -36,8 +40,8 @@
         >
       </v-row>
     </content-group>
```

Comment:
i think we need to remove disabling of btn. when there are no values. 
![image](https://user-images.githubusercontent.com/77353155/191111944-2c7fed2d-927f-4f27-bc5b-c7fceaebf14c.png)
![image](https://user-images.githubusercontent.com/77353155/191111978-235b2f88-0745-4e76-a86d-ce7e2a4f0bca.png)
line 34

Reply:
changed @ann-aot